tests/robustness/validate/patch_history.go

```diff
@@ -42,13 +42,12 @@ func patchFailedRequestWithInfiniteReturnTime(reports []report.ClientReport) []r
 	return ret
 }
 
-func patchLinearizableOperations(reports []report.ClientReport, persistedRequests []model.EtcdRequest) []porcupine.Operation {
-	allOperations := relevantOperations(reports)
+func patchLinearizableOperations(linearizableOperations []porcupine.Operation, reports []report.ClientReport, persistedRequests []model.EtcdRequest) []porcupine.Operation {
```

Comment:
change in variable name is not needed, linearizable is implied from function name

Reply:
Done